ms/MSSel/MSFieldIndex.cc

```diff
@@ -132,14 +131,15 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     else       reg=reg.fromPattern(pattern);
     
     //    cerr << "Pattern = " << pattern << "  Regex = " << reg.regexp() << endl;
-    IPosition sh(msFieldCols_p.name().getColumn().shape());
+    Vector<Bool> flagRow=msFieldCols_p.flagRow().getColumn();
+    Vector<String> codes=msFieldCols_p.code().getColumn();
+    IPosition sh(codes.shape());
```

Comment:
Does codes.shape() here have the same value that msFieldCols_p.name().getColumn().shape() had in line 135 of the old code? Actually, your code is likely better given how sh is used later, but I thought I should ask.